### DISCUSSION TITLE
Lot's of spining wheel while searching for category

### DISCUSSION THREAD

**Discussion by mr-cheffy** [community/community#46557] [Repositories]
### Select Topic Area

Bug

### Body

The video explains it all. (sorry I can't crop the video)

https://user-images.githubusercontent.com/91018726/217094779-def1b50d-3058-4e1a-ba10-5b65b2c8a468.mp4

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @mauro-balades ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by mr-cheffy** [reply]
thanks!
